modules/gdnative/godot/godot_node_path.h

```diff
@@ -35,31 +35,39 @@ extern "C" {
 #endif
 
 #include <stdint.h>
+#include <wchar.h>
 
 #ifndef GODOT_CORE_API_GODOT_NODE_PATH_TYPE_DEFINED
+#define GODOT_CORE_API_GODOT_NODE_PATH_TYPE_DEFINED
```

Comment:
I thought about simplifying this, currently all the structs are the same size as the 64 bit variants of the Godot types.
For 32 bit some of it should fall apart. So I think it's better to have a `GODOT_NODE_PATH_TYPE_SIZE` macro that specifies the size in bytes.

But this doesn't have to be done in this PR, just mentioning it here. 🙈